src/Pages/Virkopedia/VirkopediaArticle.jsx

```diff
@@ -0,0 +1,8 @@
+export const VirkopediaArticle = ({ heading, paragraphs }) => (
+  <article>
+    <h3>{heading}</h3>
+    {paragraphs.map((paragraph) => (
+      <p key={paragraph}>{paragraph}</p>
```

Comment:
here you would probably use a better key then a paragraph. Some kind of id would come in the response you could use